src/app/app/documentation/ai-chat-tools/[slug]/page.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import Link from 'next/link';
+import { useParams } from 'next/navigation';
+
+import { useAITools } from '@/services/ai-agent/tools/tools';
+
+export default function SingleChatToolPage() {
+  const { slug } = useParams();
+
+  const selectedTool = useAITools()?.find((tool) => tool.id === slug);
```

Comment:
Small thing, but I would use `toolId` instead of `slug` in the file name and here - would be more self explanatory, even when looking at directory organisation.

Reply:
As agreed, I fixed it and I told Loris to fix it in `develop`.